Demo/Application/Features/UnionPay/BraintreeDemoUnionPayViewController.m

```diff
@@ -123,7 +123,10 @@ - (void)enroll:(__unused UIButton *)button {
                 
                 self.progressBlock(@"Tokenizing card");
                 
+#pragma clang diagnostic push
```

Comment:
Should we remove the UnionPay demo soon?

Reply:
Yes! We should remove this demo when we remove UnionPay from the `next-major-version` branch 🥳